app/controllers/association/recognition/show.js

```diff
@@ -19,16 +19,17 @@ export default class AssociationRecognitionShowController extends Controller {
 
   deleteRecognition = task({ drop: true }, async (recognition) => {
     try {
-      await recognition.setProperties({
-        status: 'http://data.lblod.info/document-statuses/verwijderd',
-      });
-      await recognition.save();
+      // await recognition.setProperties({
+      //   status: 'http://data.lblod.info/document-statuses/verwijderd',
+      // });
+      await recognition.deleteRecord();
+      await recognition.save(); // HARD DELETE THE RECOGNITION
```

Comment:
as we couldn't find a proper solution to fix filtering of nested arrays in elasticsearch, I think this is reasonable. TBC with @bdevloed